### PR TITLE
Update managing-scheduled-reminders-for-your-team.md with "Get mentioned on slack" section

### PR DESCRIPTION
**Pull request by ranyitz**
### Why:

I've had to contact support to figure this out, so I thought it would be good to add it to the official docs.

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by ranyitz** [comment]
Before i'm going to invest any time fixing the lint errors, please let me know if you'd like to accept this PR first. And ideally, what should be done from your perspective

**Comment by nguyenalex836** [comment]
@ranyitz Thanks so much for opening a PR! I'll get this triaged for review ✨

**Comment by felicitymay** [comment]
Hi @ranyitz 👋🏻 

Apologies for the delay in reviewing. Thanks for flagging your difficulty in finding the information you needed to make Slack reminders work properly.

I'd like to be sure that I understand your use case properly before reviewing this PR because it's possible that we should add information about reminders in private channels to more articles than just this one.

It sounds as if you were using private Slack channels for reminders and needed to go through some additional steps before the reminders worked. Something like the steps described in the [`README`](https://github.com/integrations/slack?tab=readme-ov-file#getting-started) for the integration. Have I understood correctly, or was there more to the problem?

**Comment by ranyitz** [comment]
@felicitymay thanks for returning to me on that.

Yes we used private slack channels for reminders, i'll provide some additional context.
The reminders integration is less effective if the reminded user is not notified. In our case, some users were mentioned and thus notified, and other users were not. I couldn't find what needs to be done and approached the support:

image

Consider that the entry point for this feature is https://docs.github.com/en/organizations/organizing-members-into-teams/managing-scheduled-reminders-for-your-team and not https://github.com/integrations/slack?tab=readme-ov-file#getting-started

**Comment by felicitymay** [comment]
Thanks for confirming your use case ✨ 

I'm going to look into this a little further because it seems as there are several articles in GitHub docs where this information is relevant. It might make sense to create a [reusable](https://docs.github.com/en/contributing/writing-for-github-docs/creating-reusable-content) and then include that text in the relevant articles. 🤔 

I'll get back to you on this tomorrow.

**Review by Faphouse — COMMENTED** [review]
[](url)

**Comment by felicitymay** [comment]
@ranyitz - it seems as if this information is needed in two files:

- [Managing scheduled reminders for your organization](https://docs.github.com/en/organizations/managing-organization-settings/managing-scheduled-reminders-for-your-organization)
- [Managing scheduled reminders for your team](https://docs.github.com/en/organizations/organizing-members-into-teams/managing-scheduled-reminders-for-your-team)

The related article for users, [Managing your scheduled reminders](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/managing-your-scheduled-reminders), doesn't seem to include the option to select a Slack channel.

I made some changes locally and tested them before committing them to your branch. I've expanded the information about choosing a Slack channel to let people know that if they choose a private channel, they'll need to ask users to run extra steps. In addition, I've added links to the `README` file for the integration.

It would be good to know if this would have answered your questions.

File: data/reusables/reminders/slack-channel.md
Comment (ranyitz) [inline data/reusables/reminders/slack-channel.md:3]: Let me correct it (to the best of my knowledge)

1. Using `/invite @github` to a private channel is mandatory for the integration to work.
2. Running `/github signin` is not an alternative to running `/invite @github`, in fact the integration works and send reminders without it. Running `/github signin` for each users will allow them to get tagged as part of the reminder.

The last part about tagging of users is the missing part that need to be somewhere in the docs. Please see my original version of the PR.

**Review by ranyitz — COMMENTED** [review]
@felicitymay I added a review comment

File: data/reusables/reminders/slack-channel.md
Comment (ranyitz) [reply data/reusables/reminders/slack-channel.md:3]: When saying tagging I meant, mention. So the user gets a notification and as a result, aware that they need to review a PR.

Comment (felicitymay) [reply data/reusables/reminders/slack-channel.md]: Thank you for your clear review comments. I've updated the tip.

File: data/reusables/reminders/slack-channel.md
Comment (felicitymay) [inline data/reusables/reminders/slack-channel.md:3]: Thanks for your clear feedback. It sounds as if the `README` file for the integration might be slightly outdated 🤔 

Is this now a correct description? Or do users always need to run `/github signin` if they want to be mentioned?

**Comment by felicitymay** [comment]
I'm not sure why the preview keeps failing. I'm going to update the branch from main and see if that fixes the problem.

**Comment by felicitymay** [comment]
You can see the preview now here: [Creating a scheduled reminder for an organization](https://docs-31801-5d689c.preview.ghdocs.com/en/organizations/managing-organization-settings/managing-scheduled-reminders-for-your-organization#creating-a-scheduled-reminder-for-an-organization) and [Creating a scheduled reminder for a team](https://docs-31801-5d689c.preview.ghdocs.com/en/organizations/organizing-members-into-teams/managing-scheduled-reminders-for-your-team#creating-a-scheduled-reminder-for-a-team).

File: data/reusables/reminders/slack-channel.md
Comment (ranyitz) [inline data/reusables/reminders/slack-channel.md:3]:
```suggestion
   > If this Slack channel is private, you will need to invite the integration into the channel: `/invite @github`. In addition, you need to ask users to run `/github signin` in one of their Slack channels, otherwise they will not be @mentioned. For more information, see [Getting started](https://github.com/integrations/slack?tab=readme-ov-file#getting-started) in the Slack integrations documentation.
```

**Review by ranyitz — COMMENTED** [review]
lgtm ✅

**Review by felicitymay — APPROVED** [review]

**Comment by ranyitz** [comment]
@felicitymay I'm not able to merge this PR, so i'll leave it to you from here.

**Comment by felicitymay** [comment]
> I'm not able to merge this PR, so i'll leave it to you from here.

@ranyitz - many thanks for your collaboration on this update to the docs, and for flagging it in the first place. Once the tests have passed, I'll merge this and you'll see the changes in the live docs within a day.

**Pull request merged**
